src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -52,13 +52,14 @@ export interface ShippingState {
 }
 
 export class ShippingRoute extends Component<ShippingProps, ShippingState> {
-  // TODO: Fill in with Relay data on load.
-  // See: https://artsyproduct.atlassian.net/browse/PURCHASE-376
   state = {
-    shippingOption: "SHIP" as OrderFulfillmentType,
+    shippingOption: ((this.props.order.requestedFulfillment &&
+      this.props.order.requestedFulfillment.__typename.toUpperCase()) ||
+      "SHIP") as OrderFulfillmentType,
     address: {
       ...emptyAddress,
       country: "US",
+      ...(this.props.order.requestedFulfillment as Partial<Address>),
```

Comment:
re: moving sanitizedPropsAddress  a level up

I'm imagining simply replacing this expression with something like `_.pick(this.props.order.requestedFulfillment, Object.keys(emptyAddress))` or equivalent.

Reply:
Ah that's way better, yeah.